test/unit/script/plugin_webpack_directories_test.rb

```diff
@@ -0,0 +1,15 @@
+require 'minitest/autorun'
+require 'json'
+
+class PluginWebpackDirectoriesTest < Minitest::Test
+  def setup
+    @root_dir = File.join('../../../..', __FILE__)
+    @script = File.join("#{@root_dir}/script/plugin_webpack_directories.rb", __FILE__)
+  end
+
+  def test_bundler
+    plugin_webpack = `#{@script}`
```

Comment:
Seems that this is not producing any output in Jenkins, as it's nil in the test that is failing